django/views/debug.py

```diff
@@ -396,7 +396,10 @@ def get_traceback_data(self):
         if self.exc_type:
             c["exception_type"] = self.exc_type.__name__
         if self.exc_value:
-            c["exception_value"] = str(self.exc_value)
+            if raw_msg := getattr(self.exc_value, "raw_template_error_message", None):
+                c["exception_value"] = raw_msg
+            else:
+                c["exception_value"] = str(self.exc_value)
```

Comment:
Looks like this is equivalent?
```suggestion
            c["exception_value"] = getattr(self.exc_value, "raw_template_error_message", self.exc_value)
```